### PR TITLE
Add Build

### PR DESCRIPTION
**Pull request by frenchy64**
You can see it running on my fork: https://github.com/frenchy64/overtone/pull/6

Works on Linux/Windows/macOS! https://github.com/frenchy64/overtone/actions/runs/11428064901/job/31792900028

I think Windows should look for `scsynth.exe` on its path rather than `scsynth`. Or at least both?

Also, which versions of Clojure/Java should we test?

### CODE REVIEW

File: test/overtone/sc/core_test.clj [hunk 1]
```diff
@@ -1,48 +1,29 @@
...
+  (let [init-groups (:n-groups (status))
+        a (group :head DEFAULT-GROUP)
         b (group :head DEFAULT-GROUP)
         c (group :head DEFAULT-GROUP)]
-    (is (= 4 (:n-groups (status))))
+    (is (= 3 (- (:n-groups (status)) init-groups)))
     (run! group-free [a b c])
-    (is (= 1 (:n-groups (status))))
-    ;; We should get the old ID again with the bitset allocator
-    (is (= a (group :head DEFAULT-GROUP)))))
```
Comment (frenchy64) [inline test/overtone/sc/core_test.clj:45]: This test will never pass because we call `next-id` which always returns a fresh id.

**Comment by pfeodrippe** [comment]
@frenchy64 I was checking some of your code in this PR (great idea to use sonic-pi repo!), and got it working on the macos GH action now (using sonic-pi instead of brew) \o/

Check https://github.com/pfeodrippe/vybe/pull/3/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR214 (the last part of the ci.yml file)

Thanks for this work, man =D

**Comment by frenchy64** [comment]
@pfeodrippe it works, thanks!

**Comment by plexus** [comment]
Is this good to go in @frenchy64 ?

**Comment by frenchy64** [comment]
@plexus yep!

**Pull request merged**